src/ogrenci/entities/ogrenci.entity.ts

```diff
@@ -18,4 +19,8 @@ export class Ogrenci {
   @OneToOne(() => Bolum, { cascade: true })
   @JoinColumn()
   dept: Bolum;
+
+  @OneToOne(() => OgrenciSayac, { cascade: true })
+  @JoinColumn()
+  sayac: OgrenciSayac;
```

Comment:
`OgrenciSayac` tüm öğrencilerin sayısını tutacağı için oluşturulacak her `Ogrenci` için sadece 1 tane `OgrenciSayac` olması daha iyi olur bu yüzden burada `OneToOne` ilişki yerine `ManyToOne` kullanılsa daha iyi olabilir.

Reply:
Hatta öğrenci servisinde `OgrenciSayac` nesnesi alınıp değeri arttırılıp azaltılabilirse bu ilişkiyede gerek kalmayabilir ama geçen sefer ben `Bolum` tablosu için benzer birşey yapmaya çalıştığımda olmamıştı nedense.